src/providers/ldap/ldap_auth.c

```diff
@@ -747,16 +747,17 @@ static struct tevent_req *auth_connect_send(struct tevent_req *req)
     return subreq;
 }
 
-static void check_encryption(LDAP *ldap)
+static bool check_encryption_used(LDAP *ldap)
 {
     ber_len_t sasl_ssf = 0;
     int tls_inplace = 0;
     int ret;
 
     ret = ldap_get_option(ldap, LDAP_OPT_X_SASL_SSF, &sasl_ssf);
-    if (ret != LDAP_SUCCESS) {
+    if (ret != LDAP_OPT_SUCCESS) {
```

Comment:
Of course think this change is correct, but I would prefer if it was in a separate patch

Reply:
Done.